modules/liveConnect.js

```diff
@@ -0,0 +1,349 @@
+/**
+ * This module adds tracking support for LiveIntent's LiveConnect
+ * @module modules/liveConnect
```

Comment:
The naming convention is that these sub-modules must end in IdSystem. i.e. please change this file name to liveConnectIdSystem.js